spec/controllers/state_file/questions/md_spouse_state_id_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe StateFile::Questions::MdSpouseStateIdController do
+  describe ".show?" do
+    context "with an intake that reports married filed jointly" do
+      let!(:intake) { create :state_file_nc_intake, filing_status: :married_filing_jointly}
+
+      it "returns true" do
+        expect(described_class.show?(intake)).to eq true
+      end
+    end
+
+    context "with an intake that does not reports married filed jointly" do
+      let!(:intake) { create :state_file_nc_intake, filing_status: :single}
+
+      it "returns false" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+  end
```

Comment:
should this also have a similar `#edit` action test as well (similar to what is in `MdPrimaryStateIdController`)?